cpp/src/Slice/FileTracker.h

```diff
@@ -6,6 +6,7 @@
 #define FILE_TRACKER_H
 
 #include <IceUtil/Shared.h>
```

Comment:
Why do you keep/add these includes?

Reply:
I'm preparing a follow-up PR, with more fixes for the Parser, there is some dead code I'm getting rid of. I will include a fix for this too.